### PR TITLE
backport: trivial 2023 04 15

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
## Issue being fixed or feature implemented
Trivial backport cherry-picked via script on April 15th

## What was done?
Backports

## How Has This Been Tested?
Make -j8 check

## Breaking Changes
Should be none; did a light review

## Checklist:
  _Go over all the following points, and put an `x` in all the boxes that apply._
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

File: src/test/transaction_tests.cpp [hunk 1]
```diff
@@ -437,6 +437,33 @@ BOOST_AUTO_TEST_CASE(test_IsStandard)
     BOOST_CHECK(!IsStandardTx(CTransaction(t), reason));
     BOOST_CHECK_EQUAL(reason, "bare-multisig");
     fIsBareMultisigStd = DEFAULT_PERMIT_BAREMULTISIG;
+
+    // Check P2WPKH outputs dust threshold
```
Comment (knst) [inline src/test/transaction_tests.cpp:441]: Change a comment P2WPKH -> p2pkh?

File: src/test/transaction_tests.cpp [hunk 2]
```diff
@@ -437,6 +437,33 @@ BOOST_AUTO_TEST_CASE(test_IsStandard)
...
+    t.vout[0].scriptPubKey = CScript() << OP_0 << ParseHex("ffffffffffffffffffffffffffffffffffffffff");
+    t.vout[0].nValue = 294;
+    BOOST_CHECK(IsStandardTx(CTransaction(t), reason));
+    t.vout[0].nValue = 293;
+    BOOST_CHECK(!IsStandardTx(CTransaction(t), reason));
+    BOOST_CHECK_EQUAL(reason, "dust");
+
+    // Check P2WSH outputs dust threshold
```
Comment (knst) [inline src/test/transaction_tests.cpp:449]: same

File: src/test/transaction_tests.cpp [hunk 3]
```diff
@@ -437,6 +437,33 @@ BOOST_AUTO_TEST_CASE(test_IsStandard)
...
+    t.vout[0].scriptPubKey = CScript() << OP_0 << ParseHex("ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
+    t.vout[0].nValue = 330;
+    BOOST_CHECK(IsStandardTx(CTransaction(t), reason));
+    t.vout[0].nValue = 329;
+    BOOST_CHECK(!IsStandardTx(CTransaction(t), reason));
+    BOOST_CHECK_EQUAL(reason, "dust");
+
+    // Check future Witness Program versions dust threshold
```
Comment (knst) [inline src/test/transaction_tests.cpp:457]:
```
-// Check future Witness Program versions dust threshold
+// Check future Pay-2-Script versions dust threshold
```

**Review by knst — CHANGES_REQUESTED** [review]

**Comment by PastaPastaPasta** [comment]
@knst dropped those as the test failed :)

**Comment by UdjinM6** [comment]
linter complains `contrib/devtools/test-symbol-check.py:27:23: F821 undefined name 'get_machine'`

**Comment by PastaPastaPasta** [comment]
Resolved

**Comment by knst** [comment]
utACK

**Comment by UdjinM6** [comment]
LGTM but CI failed to start for some reason..

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**